content/gotraining.slide

```diff
@@ -713,7 +713,7 @@ Use `gofmt` or `goimport` as part of your editor/IDE setup
 - fmt
 - strings
 - time
-- encoding/json
+- [[https://goplay.space/#ywqEGKzI7T6][encoding/json]]
```

Comment:
This links to some code that is not in `samples/`, so I can't review it directly in this PR.

It contains three different TODO styles: `TDOO`, `TODO do this thing`, and `todo`. Not sure if these are notes for you or they are intended to stay. If the former, no worries. If the latter, they should be made consistent.

Also `/s/Cesar/Caesar/`

Reply:
German spelling 👅 
Done.